src/storage/v2/disk/label_property_index.cpp

```diff
@@ -0,0 +1,272 @@
+const PropertyValue kSmallestNumber = PropertyValue(-std::numeric_limits<double>::infinity());
+const PropertyValue kSmallestString = PropertyValue("");
+const PropertyValue kSmallestList = PropertyValue(std::vector<PropertyValue>());
+const PropertyValue kSmallestMap = PropertyValue(std::map<std::string, PropertyValue>());
+const PropertyValue kSmallestTemporalData =
+    PropertyValue(TemporalData{static_cast<TemporalType>(0), std::numeric_limits<int64_t>::min()});
+
+uint64_t DiskLabelPropertyIndex::ApproximateVertexCount(LabelId label, PropertyId property) const { return 10; }
+
+uint64_t DiskLabelPropertyIndex::ApproximateVertexCount(LabelId label, PropertyId property,
+                                                        const PropertyValue &value) const {
+  return 10;
+}
+
+uint64_t DiskLabelPropertyIndex::ApproximateVertexCount(LabelId label, PropertyId property,
+                                                        const std::optional<utils::Bound<PropertyValue>> &lower,
+                                                        const std::optional<utils::Bound<PropertyValue>> &upper) const {
+  return 10;
+}
+
+std::vector<std::pair<LabelId, PropertyId>> DiskLabelPropertyIndex::ClearIndexStats() {
+  std::vector<std::pair<LabelId, PropertyId>> deleted_indexes;
+  deleted_indexes.reserve(stats_.size());
+  std::transform(stats_.begin(), stats_.end(), std::back_inserter(deleted_indexes),
+                 [](const auto &elem) { return elem.first; });
+  stats_.clear();
+  return deleted_indexes;
+}
+
+std::vector<std::pair<LabelId, PropertyId>> DiskLabelPropertyIndex::DeleteIndexStatsForLabel(
+    const storage::LabelId &label) {
+  std::vector<std::pair<LabelId, PropertyId>> deleted_indexes;
+  for (auto it = stats_.cbegin(); it != stats_.cend();) {
+    if (it->first.first == label) {
+      deleted_indexes.push_back(it->first);
+      it = stats_.erase(it);
+    } else {
+      ++it;
+    }
+  }
+  return deleted_indexes;
+}
+
+void DiskLabelPropertyIndex::SetIndexStats(const storage::LabelId &label, const storage::PropertyId &property,
+                                           const IndexStats &stats) {
+  stats_[{label, property}] = stats;
+}
+
+std::optional<IndexStats> DiskLabelPropertyIndex::GetIndexStats(const storage::LabelId &label,
+                                                                const storage::PropertyId &property) const {
+  if (auto it = stats_.find({label, property}); it != stats_.end()) {
+    return it->second;
+  }
+  return {};
+}
+
+void DiskLabelPropertyIndex::LoadIndexInfo(const std::vector<std::string> &keys) {
+  for (const auto &label_property : keys) {
+    std::vector<std::string> label_property_split = utils::Split(label_property, ",");
+    index_.emplace(std::make_pair(LabelId::FromUint(std::stoull(label_property_split[0])),
+                                  PropertyId::FromUint(std::stoull(label_property_split[1]))));
```

Comment:
Do we need to add here MG_ASSERT or something? How do we know that it is always list of 2 and that converting to long long will work?